src/main/scala/edu/ie3/simona/config/ConfigFailFast.scala

```diff
@@ -431,67 +432,71 @@ case object ConfigFailFast extends LazyLogging {
     * @param refSystem
     *   the [[SimonaConfig.RefSystemConfig]] that should be checked
     */
-  private def checkRefSystem(refSystem: RefSystemConfig): Unit = {
 
-    val voltLvls =
-      refSystem.voltLvls.getOrElse(List.empty[SimonaConfig.VoltLvlConfig])
-    val gridIds = refSystem.gridIds.getOrElse(List.empty[String])
+  private def checkRefSystem(refSystems: List[RefSystemConfig]): Unit = {
+    refSystems.foreach { refSystem =>
+      {
+        val voltLvls =
+          refSystem.voltLvls.getOrElse(List.empty[SimonaConfig.VoltLvlConfig])
+        val gridIds = refSystem.gridIds.getOrElse(List.empty[String])
```

Comment:
Why do I need to pass a grid id ?
Can't we just define a default refsystem for all sensible voltage level and use that.
My goal is that the user does not even need to worry what a ref system is.

Reply:
This will check the grid id when the users provide a RefSystem, just as it does in the past. You want to remove the ability to specify a RefSystem via config completely and not just make it optional?

Reply:
whoops. you are right slightly misread it. that gets only run if a refsystem is actually defined